data/phones/lib/list_phones.py

```diff
@@ -17,12 +17,18 @@
 
 import ipapy
 
-OTHER_VALID_IPA = frozenset(
+OTHER_VALID_IPA = set(
```

Comment:
If this set is just a temporary one (i.e., isn't really used for anything but constructing `OTHER_VALID_IPA`) I probably would us `_this_kind_of_identifier` instead of `CONSTANT_IDENTIFIER`. Also, since I don't see it mutated here can it be a `frozenset`? The same thing applies to `SUFFIXED_OTHER_VALID_IPA`.

Then, one can do:

```python
OTHER_VALID_IPA = _other_valid_ipa | _suffixed_other_valid_ipa
```

which is slightly more elegant.